### PR TITLE
improve(test): Minor improvement of sqllogic test

### PR DESCRIPTION
**Pull request by leiysky**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

- Add a simple CLI tool.
- Support filtering test cases with regex

## Changelog

- Improvement

### CODE REVIEW

File: tests/logictest/main.py [hunk 1]
```diff
@@ -7,17 +7,25 @@
...
-    if disable_http_test is None:
-        http = TestHttp("http")
+import fire
+
+
+def run(pattern=".*", enable_mysql=True, enable_http=True):
+    """
+    Run tests
+    """
+    if enable_mysql:
```
Comment (ZhiHanZ) [inline tests/logictest/main.py:17]: I think we still need to push down the feature flags by some flags or environment variables

Comment (ZeaLoVe) [reply tests/logictest/main.py:17]: You can use config.py to keep default value, make sure always use the config of environment variables. All current environment variables must keep.

Comment (leiysky) [reply tests/logictest/main.py]: Ok, I just restore the env variables.

**Comment by leiysky** [comment]
@ZhiHanZ @ZeaLoVe PTAL

**Review by BohuTANG — APPROVED** [review]

**Pull request merged**